invenio_cli/helpers/commands.py

```diff
@@ -228,6 +228,15 @@ def demo(self):
         command = ['pipenv', 'run', 'invenio', 'rdm-records', 'demo']
         subprocess.run(command, check=True)
 
+    def stop(self):
+        """Stops containers."""
+        docker_helper = DockerHelper(
+            self.cli_config.get_project_shortname(),
+            local=True)
+
+        docker_helper.stop_containers()
+        click.secho('Stopped containers...', fg='green')
```

Comment:
[Full OCD]: IMO the ... are sort of an indication of something going on - in progress -. Therefore, I would use if for the messages *before* the action. e.g. "creating indexes...". 

For messages of things that *are already finished* I would not put the ... For example https://github.com/inveniosoftware/invenio-cli/blob/28a2e17ac9cba094ba018b3fb1e25105edfcaedf/invenio_cli/helpers/commands.py#L427

If you decide to change this (totally up to yo, as in, this is my OCD). Could you change the `Destroyed ...` messages too and remove the ...

```suggestion
        click.secho('Containers stopped', fg='green')
```